johnhenry/johnhenry.github.io
# Update astro to 1.0.0-beta.5

Update to 1.0.0-beta.5 from 0.25.
- Install latest version of astro
- Use frontmatter property to access frontmatter of md files
- Replace Astro. fetchContent with Astro.glob
- Replace Astro.request.params with Astro.params
- Ensure unprocessed scripts have the "is:inline" attribute
See: https://docs.astro.build/en/migrate/
